### PR TITLE
Custom tip for dex screener

### PR DESCRIPTION
**Pull request by neikop**
- Allow partner-swap can custom fee by params
- Fix bug that partner-swap didn't set default slippage when swapping stable coins

### CODE REVIEW

File: src/components/FeeControlGroup/index.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  return (
+    <Flex
+      sx={{
+        flexDirection: 'column',
+        width: '100%',
+        padding: '0 8px',
+      }}
+    >
+      <Text color={theme.subText} fontSize={12} fontWeight={500}>
+        Tip :
```
Comment (viet-nv) [inline src/components/FeeControlGroup/index.tsx:83]: `<Trans>Tip</Trans>:`

File: src/components/FeeControlGroup/index.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    flex: 0 0 15%;
+  }
+`
+
+const FeeControlGroup = () => {
+  const theme = useTheme()
+  const { feeAmount, enableTip } = useGetFeeConfig() ?? {}
+  const [searchParams, setSearchParams] = useSearchParams()
+
+  const [feeValue, setFee] = useState(Math.round(Number.parseFloat(feeAmount ?? '0')) || 0)
```
Comment (viet-nv) [inline src/components/FeeControlGroup/index.tsx:60]: why do we need this state? 
can use searchParams.get('feeAmount') directly.

File: src/components/FeeControlGroup/index.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      >
+        {DEFAULT_TIPS.map(tip => (
+          <DefaultFeeOption
+            key={tip}
+            onClick={() => {
+              setFee(tip)
+            }}
+            data-active={tip === feeValue}
+          >
+            {tip ? `${tip / 100}%` : 'No tip'}
```
Comment (nguyenhoaidanh) [inline src/components/FeeControlGroup/index.tsx:109]: <Trans>No tip</Trans>

File: src/components/FeeControlGroup/index.tsx [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+        flexDirection: 'column',
+        width: '100%',
+        padding: '0 8px',
+      }}
+    >
+      <Text color={theme.subText} fontSize={12} fontWeight={500}>
+        Tip :
+      </Text>
+      <Text color={theme.text3} fontSize={12} fontWeight={500}>
+        No hidden fees - Your optional tips support DEX Screener!
```
Comment (nguyenhoaidanh) [inline src/components/FeeControlGroup/index.tsx:86]: Trans

File: src/components/FeeControlGroup/index.tsx (hunk 2 above)
Comment (neikop) [reply src/components/FeeControlGroup/index.tsx]: Updated

**Review by viet-nv — APPROVED** [review]

**Pull request merged**